tokio/src/io/driver/scheduled_io.rs

```diff
@@ -56,6 +56,7 @@ impl Default for ScheduledIo {
 }
 
 impl ScheduledIo {
+    #[cfg(all(test, loom))]
```

Comment:
Hmm, if this is only used in tests now, I wonder if there's anything we can simplify based on the assumption that all readiness ops are RMWs? May be worth looking into in a follow-up...